File: Latihan b/cpp/product.cpp
// Include
#include <iostream>
#include <string>
using namespace std;

// deklarasi kelas produk
class product{
    // variabel privat
    private:
        int price;
        int idproduct;
    // method public dan konstruktor
    public:
        product(){

        }
        // get dans set semua variabel
        void setprice(int price){
            this->price = price;
        }
        int getprice(){
            return this->price;
        }
        void setidproduct(int idproduct){
            this->idproduct = idproduct;
        }
        int getidproduct(){
            return this->idproduct;
        }
        // desktruktor
        ~product(){
        }
};

File: Latihan b/Python/product.py
# deklarasi kelas produk
class product:
    # method dan konstruktor
    def __init__(self):
        self.price = 0
        self.idproduct = 0
    #  get dans set semua variabel
    def setprice(self, price):
        self.price = price
    def getprice(self):
        return self.price
    def setidproduct(self, idproduct):
        self.idproduct = idproduct
    def getidproduct(self):
        return self.idproduct